homeassistant/components/samsungtv/bridge.py

```diff
@@ -240,7 +240,8 @@ def _get_remote(self, avoid_open: bool = False):
 
     def _send_key(self, key):
         """Send the key using legacy protocol."""
-        self._get_remote().control(key)
+        if remote := self._get_remote():
```

Comment:
I think that it's better to catch the exception. That way if the TV is on but we don't know (yet), we still are able to deliver the key.

Reply:
Talked to Simone, there is no remote object so nothing to "attempt" to send to.

Reply:
Shouldn't it raise an exception then? We where talking about that the other day right. Otherwise automations can't detect failures?